features/claim_based_routing.md

```diff
@@ -0,0 +1,235 @@
+
+[//]: # (All information in this section is mandatory for promotion. Please modify the links in this)
+[//]: # (section.)
+
+- [ ] [User stories](insert_your_link_here) reviewed in a work group meeting.
+
+[//]: # (User stories are a way to communicate user value. User stories follow the style)
+[//]: # (as a [type of user], I want [an action] so that [a benefit/a value]. Istio currently has no user)
+[//]: # (story template. Maybe you can make one?)
+
+[//]: # (User stories must be presented in a work group meeting. They need no approval and are later integrated)
+[//]: # (into the RFCs, which do need approval for alpha. You may find value to negotiate within the work group where the)
+[//]: # (user stories are presented to help clarify the user stories.)
+
+- [ ] [RFC Authored] - [create an RFC using template](https://docs.google.com/document/d/1ewJoCcw5-04crH-M0xw4zFxz1cfwVCPnNyW4K3m4Yyc/template/preview).
+
+[//]: # (An RFC is mandatory to graduate to experimental. The RFC does not have to be reviewed in a work group)
+[//]: # (meeting to graduate to experimental.)
+
+- [ ] [Documentation](insert_your_link_here) for enabling and using the feature.
+
+[//]: # (The documentation instructions may exist on the developer wiki or the team drive. They may include instructions)
+[//]: # (for building running a `istioctl experimental command`, or using the preview profile,)
+[//]: # (or any other relevant information.)
+
+- [ ] [Feedback plan](insert_your_link_here).
+
+[//]: # (This may include user feedback meetings, discuss.istio.io conversations, GitHub issues, or mailing lists.)
+
+- [ ] Disabled by default.
+
+- [ ] No impact on performance when the feature is disabled.
+
+
+[//]: # (Once all other items are completed, features.yaml should be updated to promote the feature)
+
+- [ ] [features.yaml](https://github.com/istio/enhancements/blob/master/features.yaml) updated for this feature
+---
+
+## Alpha
+
+### Requirements: 
+
+**Design**
+
+- [x] RFC has been approved describing the intention of the feature as well as the user stories behind the feature. 
+
+> Design doc already approved: https://docs.google.com/document/d/1s8Lykf8-KUZTLiG4YMSCHvxBDwZzHlrrHyZ7cJymc_8)
+
+**Config**
+
+- [x] Explicit user action is required to enable this feature (e.g. a config field, config resource, or installation action). 
+
+> The user needs to modify the virtual service to enable this feature - https://istio.io/latest/docs/tasks/security/authentication/jwt-route/
+
+**Docs**
+
+- [x] Reference docs are published to preliminary.istio.io or the Istio wiki.
+
+> The task page of the feature is published at https://istio.io/latest/docs/tasks/security/authentication/jwt-route/
+
+- [x] Basic feature docs are published on preliminary.istio.io describing what the feature does, how to use it, and any caveats. 
+
+> A task has been added
+
+- [x] Release notes entries added as appropriate
+
+> Targeting for 1.16 release, will add it later when required
```

Comment:
You could point to the PR that has the RN.

Reply:
sure, will add it once I add 1.16 release notes.